services/voting-node/poetry.toml

```diff
@@ -4,7 +4,7 @@ prefer-active-python = true
 
 [virtualenvs.options]
 no-pip = true
-no-setuptools = true
+#no-setuptools = true
```

Comment:
Is it intentional ?

Reply:
It is until I work out why the voting node won't build for me.
I thought this might be causing it,  it's doesn't seem to, so I may revert it in a later push.